substrate/client/transaction-pool/tests/zombienet/limits_30.rs

```diff
@@ -0,0 +1,164 @@
+use std::{path::PathBuf, time::SystemTime};
+
+use zombienet_sdk::{
+	LocalFileSystem, Network as ZNetwork, NetworkConfig, NetworkConfigBuilder, NetworkConfigExt,
+};
+
+use super::{
+	Network, Node, ParachainConfig, RelaychainConfig, DEFAULT_BASE_DIR, DEFAULT_PC_NODE_RPC_PORT,
+	DEFAULT_RC_NODE_RPC_PORT,
+};
+use anyhow::anyhow;
+use which::which;
+
+// A zombienet network with two relaychain 'polkadot' validators and one parachain
+// validator based on yap-westend-live-2022 chain spec.
+pub struct Limits30Network {
```

Comment:
One more thought:

We could have exactly the same struct for all the networks, and just have different values for different networks.
(This struct seems to be copied all over the files).

So maybe sth like this:
```
Network::new_limits_30_network()
Network::new_single_collator()
Network::new_old_pool_small()
...
```

Reply:
Exactly! I have some TODOs already in `zombienet/mod.rs` after I realized this: https://github.com/iulianbarbu/polkadot-sdk/blob/21fda5fd98504cccfa9c0a087ddc466db8da2880/substrate/client/transaction-pool/tests/zombienet/mod.rs#L37C1-L43C68. I think they refer to the same thing.